quipucords/scanner/satellite/six.py

```diff
@@ -332,6 +291,113 @@ def host_subscriptions(scan_task, url, org_id, host_id, ssl_cert_verify=None,
     return subs_dict
 
 
+def log_message(message, inspect_task_id, scan_type,
```

Comment:
I would put this in the task method and have the task use this for all of it's logging.